app/models/profile.rb

```diff
@@ -33,6 +38,9 @@ def self.refresh_attributes!
     return unless Database.table_available?("profiles")
 
     ProfileField.find_each do |field|
+      # Don't generator accessors for static fields stored on the table.
+      next if STATIC_FIELDS.any?(field.attribute_name)
```

Comment:
Is this intended to be temporary? If not I think it may be better to disallow creating these in the first place, with a validation that checks the field's attribute_name against `STATIC_FIELDS` (or the class method if we go with replacing it).

Reply:
As I understand it, this can only go away if once all of the data has been migrated AND the attributes are deleted from `Profile.data`. So when that happens we can remove this?

I added a TODO comment.

Reply:
Thanks for the comment! I guess technically we remove it as soon as the fields themselves have been deleted, the data will eventually clean itself up (data for non-existing fields will be removed on the next save).